### ISSUE TITLE
[Submit add-on]: Mouse Enhancement v0.8.0

### ISSUE THREAD

**Issue by hwf1324** [nvaccess/addon-datastore#6425]
### Download URL

https://github.com/hwf1324/mouseEnhancement/releases/download/v0.8.0/mouseEnhancement-0.8.0.nvda-addon

### Source URL

https://github.com/hwf1324/mouseEnhancement

### Publisher

hwf1324

### Channel

stable

### License Name

GPL v2

### License URL

https://www.gnu.org/licenses/gpl-2.0.html

**Comment by seanbudd** [comment]
Hi @hwf1324 ,

Your addon manifest is invalid.
Have you tested your add-on with NVDA? I believe it would fail to load due to manifest syntax errors.

```
  File "D:\a\addon-datastore\addon-datastore\validation\_validate\createJson.py", line 207, in main
    manifest = getAddonManifest(args.file)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\a\addon-datastore\addon-datastore\validation\_validate\manifestLoader.py", line 32, in getAddonManifest
    raise err
  File "D:\a\addon-datastore\addon-datastore\validation\_validate\manifestLoader.py", line 29, in getAddonManifest
    manifest = AddonManifest(filePath)
               ^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\a\addon-datastore\addon-datastore\validation\_validate\addonManifest.py", line 77, in __init__
    super().__init__(
  File "D:\a\addon-datastore\addon-datastore\validation\.venv\Lib\site-packages\configobj\__init__.py", line 1204, in __init__
    self._load(infile, configspec)
  File "D:\a\addon-datastore\addon-datastore\validation\.venv\Lib\site-packages\configobj\__init__.py", line 1301, in _load
    raise error
configobj.ConfigObjError: Parsing failed with 3 errors.
    Invalid line ('* Update add-on template.') (matched as neither section nor keyword) at line 15.
    Invalid line ('* Add Ukrainian locale and doc. (#10 @balaraz)') (matched as neither section nor keyword) at line 16.
    Invalid line ('Thanks to @balaraz for your contribution.') (matched as neither section nor keyword) at line 18.
```

**Comment by hwf1324** [comment]
The reason is that I used the new `addon_changelog` manifest item. I will remove it and resubmit.
